sdk/Trace/Test/SpanData.php

```diff
@@ -0,0 +1,275 @@
+<?php
+
+declare(strict_types=1);
+
+namespace OpenTelemetry\Sdk\Trace\Test;
+
+use OpenTelemetry\Sdk\InstrumentationLibrary;
+use OpenTelemetry\Sdk\Resource\ResourceInfo;
+use OpenTelemetry\Sdk\Trace as SDK;
+use OpenTelemetry\Sdk\Trace\StatusData;
+use OpenTelemetry\Trace as API;
+
+class SpanData implements SDK\SpanData
```

Comment:
This is a mutable version of `SpanData`, mainly useful in tests as a mini builder so you don't _have_ to provide _every_ span constructor argument.

Reply:
Should probably do some composer something or other so this doesn't get shipped when installing the package. I.e. should be a development type only.

Reply:
You might need to move it into its own separate file and ignore it if that's what you'd like to do.  I'm sure there is a way to do it for a particular class but I don't know that answer off the top of my head.